test/fuzz/basic.ml

```diff
@@ -138,6 +142,16 @@ let iunop_64 : text instr gen =
   let+ iunop in
   I_unop (S64, iunop)
 
+(* TODO: check comment above *)
+let iunop_popcnt_32 : text instr gen =
+  let+ popcnt = const Popcnt in
+  I_unop (S32, popcnt)
+
+(* TODO: check comment above *)
+let iunop_popcnt_64 : text instr gen =
+  let+ popcnt = const Popcnt in
+  I_unop (S64, popcnt)
+
```

Comment:
These two functions could be removed I guess.

Reply:
same as above ...

Reply:
same answer :)